app/Models/Context.php

```diff
@@ -94,6 +95,13 @@ public static function currentGet($array = false) {
 	}
 
 	/**
+	 * Return true iif the current requests target a feed and not a category or all articles.
```

Comment:
It was intentional :-)
It is the abbreviation of "if and only if". Otherwise, one must write "or false otherwise" or something similar.
https://en.wikipedia.org/wiki/Iff